go/host/host.go

```diff
@@ -514,14 +508,6 @@ func (h *host) processL1Block(block *types.Block, isLatestBlock bool) error {
 		return nil
 	}
 
-	if h.genesisRequired {
```

Comment:
The sequencer now produces the genesis automatically upon receiving an L1 block and noticing no head batch exists.